packages/runtime/src/widgets/Form/FormItem.tsx

```diff
@@ -39,7 +39,7 @@ export const EnsembleFormItem: React.FC<EnsembleFormItemProps<unknown>> = (
           </label>
         ) : null
       }
-      name={formInstance ? values?.id ?? values?.label : undefined}
+      name={formInstance ? values?.label ?? values?.id : undefined}
```

Comment:
Does this regress how forms are referred to in the formValues? Can you still refer to them by form.id?

Reply:
Yeah I think this does regress this - you need to modify the error handling only.